test/src/test/java/org/corfudb/integration/ServerRestartTest.java

```diff
@@ -1,160 +1,73 @@
-    private static final String CORFU_PROJECT_DIR = new File("..").getAbsolutePath() + File.separator;
-    private static final String CORFU_CONSOLELOG = CORFU_LOG_PATH + File.separator + "consolelog";
-    private static final String KILL_COMMAND = "kill -9 ";
-    private static final String KILL_CORFUSERVER = "jps | grep CorfuServer|awk '{print $1}'| xargs kill -9";
-
-    public ServerRestartTest() {
-        CorfuRuntime.overrideGetRouterFunction = null;
-    }
-
-    public static void setUp() throws Exception {
-        FileUtils.cleanDirectory(new File(CORFU_LOG_PATH));
-    }
-
-    private static Process runCorfuServer() throws IOException, InterruptedException {
-        ProcessBuilder builder = new ProcessBuilder();
-        builder.command("sh", "-c", CORFU_SERVER_COMMAND);
-        builder.directory(new File(CORFU_PROJECT_DIR));
-        Process corfuServerProcess = builder.start();
-        StreamGobbler streamGobbler = new StreamGobbler(corfuServerProcess.getInputStream(), CORFU_CONSOLELOG);
-        Executors.newSingleThreadExecutor().submit(streamGobbler);
-        return corfuServerProcess;
-    }
-
-    private static boolean shutdownCorfuServer(Process corfuServerProcess) throws IOException, InterruptedException {
-        long pid = getPid(corfuServerProcess);
-        ProcessBuilder builder = new ProcessBuilder();
-        builder.command("sh", "-c", KILL_COMMAND + pid);
-        Process p = builder.start();
-        p.waitFor();
-        builder = new ProcessBuilder();
-        builder.command("sh", "-c", KILL_CORFUSERVER);
-        p = builder.start();
-        p.waitFor();
-        return true;
-    }
-
-    private static long getPid(Process p) {
-        long pid = -1;
-
-        try {
-            if (p.getClass().getName().equals("java.lang.UNIXProcess")) {
-                Field f = p.getClass().getDeclaredField("pid");
-                f.setAccessible(true);
-                pid = f.getLong(p);
-                f.setAccessible(false);
-            }
-        } catch (Exception e) {
-            pid = -1;
-        }
-        return pid;
-    }
-
-    private static CorfuRuntime createRuntime() {
-        CorfuRuntime rt = new CorfuRuntime(TEST_ENDPOINT)
-                .connect();
-        return rt;
-
-    }
-
-    private static Map<String, Integer> createMap(CorfuRuntime rt) {
-        Map<String, Integer> map = rt.getObjectsView()
-                .build()
-                .setStreamName("A")
-                .setType(SMRMap.class)
-                .open();
-        return map;
-    }
+public class ServerRestartTest extends AbstractIntegrationTest {
 
     /**
      * Test recovery with a failed over server and a new client.
      *
      * @throws Exception
      */
     @Test
-    public void testSequencerRecoveryFailoverServerAndClient() throws Exception {
-        setUp();
+    public void testRecoveryFailoverServerAndClient() throws Exception {
```

Comment:
It seems like all these tests are just a subset of testRandomizedRecovery. Can you clobber them?